README.rst

```diff
@@ -14,3 +14,22 @@ It is not / and should not be published on PyPi.
 
 Also tests are designed to run from Chevah VPN with access to staging Trac
 instance.
+
+
+serverless
+==========
+
+To run offline for testing purposes:
+
+```sh
+npm install  # Required after changing `serverless.yml`.
+sls offline
```

Comment:
I think that is best to use `npm install -g` from a nodeenv ...and use `serverless` offline. to reduce the confusion with the `sls` command

and I think that the readme should be updated to instruct that `pip install .` should be called before `servereless offline` to get the packages  and prepare the deployment environment

Reply:
I replaced `sls` with `serverless` everywhere. I used `poetry install` instead of `pip install .`, but `pip` should work also.